reactive/health/src/main/java/io/helidon/reactive/health/HealthSupport.java

```diff
@@ -113,7 +113,8 @@ private HealthSupport(Builder builder) {
         }
 
         // Lazy values to prevent early init of maybe-not-yet-configured FT thread pools
-        this.timeout = LazyValue.create(() -> Timeout.create(Duration.ofMillis(builder.timeoutMillis)));
+        this.timeout = LazyValue.create(() -> Timeout.create(
```

Comment:
Also this is actually a bug in the original code, this should be:
```java
Duration timeoutDuration = builder.timeout;
this.timeout = LazyValue.create(() -> Timeout.create(timeoutDuration));
```
Otherwise we would use timeout instance on the builder available when we execute the supplier (which may differ from the value at build time)